packages/web-components/src/drawer/drawer.styles.ts

```diff
@@ -0,0 +1,191 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/index.js';
+import {
+  colorBackgroundOverlay,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  colorTransparentStroke,
+  curveAccelerateMid,
+  curveDecelerateMid,
+  curveLinear,
+  durationGentle,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  shadow64,
+  strokeWidthThick,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+/** Drawer styles
+ * @public
+ */
+export const styles = css`
+  ${display('block')}
+
+  :host {
+    --dialog-backdrop: ${colorBackgroundOverlay};
+    overflow: hidden;
+  }
+
+  dialog {
+    display: none;
+    z-index: var(--drawer-elevation, 1000);
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    font-family: ${fontFamilyBase};
+    font-weight: ${fontWeightRegular};
+    color: ${colorNeutralForeground1};
+    max-width: var(--drawer-width, 592px);
+    max-height: 100vh;
+    overflow: hidden;
```

Comment:
I found relying on `dialog`'s default overflow a lot easier to work with with rather than casting that down the `*-body` component.